### DISCUSSION TITLE
Problem with q-select in popup + filtering/option issues

### DISCUSSION THREAD

**Discussion by DaveDisco** [quasarframework/quasar#16576] [General - Components / Directives / etc] [answered]
### What happened?

Issues with filtered q-select which allows new values to be added, running in a popup menu:

1. A new option can be added by typing in a value then entering return. However, if you click on the Set instead of pressing return the value is not added.

2. If you click the popup again to add a new value, the original value remains displayed as you type in the value. This is off putting. What you might really want is that when you start editing, the displayed value is copied into the value that you can then edit.

### What did you expect to happen?

1. The new value is added on Set
2. I can add an option by editing an exsting one. 

### Reproduction URL

https://codepen.io/Skadave/pen/ExrmmYY

### How to reproduce?

Click on popup button

### Flavour

Quasar CLI with Vite (@quasar/cli | @quasar/app-vite)

### Areas

Components (quasar)

### Platforms/Browsers

Chrome

**Comment by pdanpdan** [comment, marked as answer]
https://codepen.io/pdanpdan/pen/rNPmvRR?editors=1010

**Reply by DaveDisco** [reply]
Thanks. That seems to be perfect. I think it would have taken me a long time to work out (stumble  across) that particular combination. Maybe those settings could be in the docs example?

**Reply by DaveDisco** [reply]
I've found a problem with this solution. It does not work if you enter a value and then very quickly click Set. I presume the click event beats the blur event so the lastFilterText is not updated

**Reply by pdanpdan** [reply]
That is because of the input debounce - set it to 0
Updated the codepen

**Reply by DaveDisco** [reply]
that works, thanks
